StarkInfra/StarkInfra/PixChargeback/PixChargeback.cs

```diff
@@ -208,11 +216,13 @@ public static PixChargeback Get(string id, User user = null)
         /// <br/>
         /// Parameters (optional):
         /// <list>
-        ///     <item>status [string, default null]: filter for status of retrieved objects. ex: "created", "failed", "delivered", "closed", "canceled"</item>
+        ///     <item>limit [integer, default null]: maximum number of objects to be retrieved. Unlimited if null. ex: 35</item>
         ///     <item>after [DateTime or string, default null]: date filter for objects created only after specified date. ex: DateTime(2020, 3, 10)</item>
         ///     <item>before [DateTime or string, default null]: date filter for objects created only before specified date. ex: DateTime(2020, 3, 10)</item>
-        ///     <item>ids [list of strings, default null]: list of ids to filter retrieved objects. ex: new List<string>{ "5656565656565656", "4545454545454545"]</item>
-        ///     <item>limit [integer, default null]: maximum number of objects to be retrieved. Unlimited if null. ex: 35</item>
+        ///     <item>status [string, default null]: filter for status of retrieved objects. Options: "created", "failed", "delivered", "closed", "canceled"</item>
```

Comment:
Me chama pra esse comentário aqui pq eu vi que ta meio inconsciente pra saber se esse status é uma list of string (como ta em Python) ou string mesmo.